File: modul2/Python/PRAK204-2310817210029-Putra_Whyra_Pratama_Setiawan.py
pi = 3.142857
jarijari, tinggi = input("").split(" ")
volume = float(round(pi*int(jarijari)*int(jarijari)*int(tinggi), 2))
luas = float(round(2*pi*int(jarijari)*int(jarijari)+2*pi*int(jarijari)*int(tinggi), 2))
keliling = float(round(pi*2*int(jarijari), 2))
print("Volume = %.2f"% (volume))
print("Luas = %.2f"% (luas))
print("Keliling = %.2f"% (keliling))

File: modul2/C/PRAK204-2310817210029-Putra_Whyra_Pratama_Setiawan.c
#include <stdio.h>
#include <math.h>
void main() {
    float jarijari, tinggi, pi=3.142857;
    scanf("%f %f", &jarijari, &tinggi);
    printf("Volume=%.2f\n", (pi*jarijari*jarijari*tinggi));
    printf("Luas=%.2f\n", (2*pi*(jarijari*jarijari)+2*pi*jarijari*tinggi));
    printf("Keliling=%.2f\n", (pi*2*jarijari));
}